receiver/sqlserverreceiver/metadata.yaml

```diff
@@ -89,13 +89,17 @@ metrics:
     enabled: true
     description: Total number of transaction log expansions for a database.
     unit: "{growths}"
-    gauge:
+    sum:
+      monotonic: true
+      aggregation: cumulative
       value_type: double
```

Comment:
@bogdandrutu it's the way it is presented from the windows performance counters. it might be able to be changed to an int with some research, but i'd say that's outside the scope of this bug fix.

Reply:
@StefanKurek will you please create an issue to track this?